QuickTour/QuickTour.tex

```diff
@@ -261,6 +261,10 @@ \section{Getting started}
 \caption{The Preference Browser.\label{fig:prefBrowser}}
 \end{figure}
 
+Depending on the resolution of your display and the scaling mechanism of your operating system, you might encounter very small or large fonts when you open \sq for the first time.

```

Comment:
Is this up to date? Squeak will automatically detect your scale factor and even if the detection is wrong, the preference wizard will provide you with a prominent list of scale factors on the first start. Should this be mentioned?

Reply:
Fair, I am being very conservative here. I will soften the description to something like "you might want to change the initial scale \sq chose for you". Will also add the PreferenceWizard. :)